[2025-07-12T15:51:44.830+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:51:42.609694+00:00 [queued]>
[2025-07-12T15:51:44.834+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:51:42.609694+00:00 [queued]>
[2025-07-12T15:51:44.834+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-12T15:51:44.839+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): detect_flight_anomalies> on 2025-07-12 15:51:42.609694+00:00
[2025-07-12T15:51:44.842+0000] {standard_task_runner.py:60} INFO - Started process 345 to run task
[2025-07-12T15:51:44.844+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_ml_dag', 'detect_flight_anomalies', 'manual__2025-07-12T15:51:42.609694+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/flight_ml_dag.py', '--cfg-path', '/tmp/tmpf9z1vi0z']
[2025-07-12T15:51:44.845+0000] {standard_task_runner.py:88} INFO - Job 150: Subtask detect_flight_anomalies
[2025-07-12T15:51:44.865+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:51:42.609694+00:00 [running]> on host f1b3ce86e5d1
[2025-07-12T15:51:44.895+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_scientist' AIRFLOW_CTX_DAG_ID='flight_ml_dag' AIRFLOW_CTX_TASK_ID='detect_flight_anomalies' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T15:51:42.609694+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-12T15:51:42.609694+00:00'
[2025-07-12T15:51:44.898+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-12T15:51:44.928+0000] {logging_mixin.py:188} INFO - ‚úÖ Anomaly detection completed
[2025-07-12T15:51:44.928+0000] {logging_mixin.py:188} INFO - üö® Total anomalies found: 179
[2025-07-12T15:51:44.928+0000] {logging_mixin.py:188} INFO - ‚è±Ô∏è Duration anomalies: 0
[2025-07-12T15:51:44.928+0000] {logging_mixin.py:188} INFO - üìÖ Scheduling anomalies: 82
[2025-07-12T15:51:44.929+0000] {logging_mixin.py:188} INFO - üìä Performance anomalies: 97
[2025-07-12T15:51:44.929+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-12T15:51:44.932+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_ml_dag, task_id=detect_flight_anomalies, execution_date=20250712T155142, start_date=20250712T155144, end_date=20250712T155144
[2025-07-12T15:51:44.942+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-12T15:51:44.951+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
